package.json

```diff
@@ -33,7 +33,7 @@
     "redux": "^4.1.0",
     "redux-promise": "^0.6.0",
     "redux-thunk": "^2.3.0",
-    "sariska-media-transport": "https://github.com/SariskaIO/sariska-media-transport.git#v2.0.55",
+    "sariska-media-transport": "https://github.com/SariskaIO/sariska-media-transport.git#v2.0.58",
```

Comment:
this should be "sariska-media-transport": "https://github.com/SariskaIO/sariska-media-transport.git#2.0.58",